README.rst

```diff
@@ -41,9 +41,9 @@ well. On Debian-based systems::
 
   $ sudo apt-get install bison
   ...
-  $ touch src/wasm-parser.y
+  $ touch src/wasm-bison-parser.y
```

Comment:
There were some errors in the README file that I discovered as I was doing the port. I figured I might as well put the change in this PR since it's a small change.